dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/imports/bundle/TrackerImporterServiceTest.java

```diff
@@ -85,6 +86,8 @@ public void setUp() {
         new DefaultTrackerImportService(
             trackerBundleService, validationService, trackerPreprocessService);
 
+    injectSecurityContext(user);
```

Comment:
I think we added a static version specifically so mock based tests do not have to extend `TestBase` for this

Reply:
Done